src/screens/settings/Settings.d.ts

```
@@ -0,0 +1,168 @@
+export type ColorPickerSetting =
+  | _ColorPickerSetting<'MMKV'>
+  | _ColorPickerSetting<undefined>;
+
+export type ReaderThemeSetting = { type: 'ReaderTheme' };
+export type ReaderTTSSetting = { type: 'TTS' };
+export type RepoSetting = { type: 'Repo' };
+export type TrackerSetting = {
+  type: 'Tracker';
+  trackerName: 'AniList' | 'MyAnimeList';
+};
+export type InfoItem = { type: 'InfoItem'; title: string };
```

Biome (2.1.2)

[error] Shouldn't redeclare 'InfoItem'. Consider to delete it or rename it.

'InfoItem' is defined here:


(lint/suspicious/noRedeclare)